openvino_tensorflow/api.cc

```diff
@@ -101,6 +102,8 @@ extern const char* get_disabled_ops() {
 void enable_dynamic_fallback() { EnableDynamicFallback(); }
 void disable_dynamic_fallback() { DisableDynamicFallback(); }
 
+void disable_rewrite_pass() { DisableRewritePass(); }
```

Comment:
As far as I know, grappler pass is not enabled for OVTF this release. So, what happens if the rewrite pass is disabling?

Reply:
This was added for enabling grappler pass, so when grappler pass optimize APIs are called then rewrite pass is disabled using this API. This is for internal use and not documented for users to use it.